app/views/layouts/admin.html.erb

```diff
@@ -9,10 +9,20 @@
 <% end %>
 
 <% content_for :head do %>
+  <%= render "govuk_publishing_components/components/google_tag_manager_script", {
+    gtm_id: "GTM-P93SHJ4Z"
```

Comment:
It'd be good if we could set this with an environment variable - then we'd not have the analytics running in people's local environments and could, if we chose to, have different analytics for non-prod environments like staging and integration to not pollute prod data (would save a whole bunch of filtering).

Would also mean we don't need to duplicate the key as well in separate files.

Reply:
Here's an example from content publisher code: https://github.com/alphagov/content-publisher/blob/6fb72ab4038ae3150a777364ed51dfeadeaca47f/app/views/layouts/application.html.erb#L16-L22

One of the interesting things from Content Publisher, that you may also have to face and potentially resolve is also that some datalayer stuff is configured before the component (which seems a bit in-elegant): https://github.com/alphagov/content-publisher/blob/6fb72ab4038ae3150a777364ed51dfeadeaca47f/app/views/layouts/application.html.erb#L7-L15 - which if we have to do similar we might want to extend the component.

Reply:
Good point! Have updated to match content publisher and added a PR to set the environmental variable
https://github.com/alphagov/govuk-helm-charts/pull/1243